packages/desktop/views/dashboard/wallet/views/multi-step/MultiStep.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import { ImplicitAccountCreationState } from '@contexts/implicit-account-creation'
+
+    const IMPLICIT_ACCOUNT_STEPS: ImplicitAccountCreationState[] = Object.values(ImplicitAccountCreationState)
+
+    let activeState: ImplicitAccountCreationState | null = null
+    let onNext: () => Promise<boolean>
+    let currentStep: number = 0
+    let activeComponent
+
+    async function loadComponent() {
```

Comment:
we could probably do this like in https://github.com/iotaledger/firefly/blob/161075fa61297479e681e841a57d1fd41b0786bd/packages/desktop/views/dashboard/Dashboard.svelte#L38-L39